ocw/lib/azure.py

```diff
@@ -100,126 +99,44 @@ def list_disks_by_resource_group(self, resource_group):
-        for img in container_client.list_blobs():
-            m = self.parse_image_name(img.name)
-            if m:
-                self.log_dbg('Blob {} is candidate for deletion with build {} ', img.name, m['build'])
-
-                if img.name not in keep_images:
-                    self.log_info("Delete blob '{}'", img.name)
-                    if self.dry_run:
-                        self.log_info("Deletion of blob image {} skipped due to dry run mode", img.name)
-                    else:
-                        container_client.delete_blob(img.name, delete_snapshots="include")
+            if 'pcw_ignore' not in c['metadata']:
+                self.log_dbg('Found container {}', c.name)
+                container_blobs = self.container_client(c.name).list_blobs()
+                for blob in container_blobs:
+                    if (self.older_than_max_age_hours(blob.last_modified)):
+                        if self.dry_run:
+                            self.log_info("Deletion of blob {} skipped due to dry run mode", blob.name)
+                        else:
+                            self.log_info("Deleting blob {}", blob.name)
+                            self.container_client(c.name).delete_blob(blob.name, delete_snapshots="include")
 
-    def cleanup_images_from_rg(self, keep_images):
+    def cleanup_images_from_rg(self):
         for item in self.list_images_by_resource_group(self.__resource_group):
-            m = self.parse_image_name(item.name)
-            if m:
-                self.log_dbg('Image {} is candidate for deletion with build {} ', item.name, m['build'])
-                if item.name not in keep_images:
+            if self.older_than_max_age_hours(item.changed_time):
+                if self.dry_run:
+                    self.log_info("Deletion of image {} skipped due to dry run mode", item.name)
+                else:
                     self.log_info("Delete image '{}'", item.name)
-                    if self.dry_run:
-                        self.log_info("Deletion of image {} skipped due to dry run mode", item.name)
-                    else:
-                        self.compute_mgmt_client().images.begin_delete(self.__resource_group, item.name)
+                    self.compute_mgmt_client().images.begin_delete(self.__resource_group, item.name)
 
-    def cleanup_disks_from_rg(self, keep_images):
+    def cleanup_disks_from_rg(self):
         for item in self.list_disks_by_resource_group(self.__resource_group):
-            m = self.parse_image_name(item.name)
-            if m:
-                self.log_dbg('Disk {} is candidate for deletion with build {} ', item.name, m['build'])
-
-                if item.name not in keep_images:
-                    if self.compute_mgmt_client().disks.get(self.__resource_group, item.name).managed_by:
-                        self.log_warn("Disk is in use - unable delete {}", item.name)
+            if self.older_than_max_age_hours(item.changed_time):
```

Comment:
```suggestion
            if self.is_outdated(item.changed_time):
```

I think that name would be more descriptive, WDYT?

Reply:
done